packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/ErrorView.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { settings } from 'carbon-components';
+import { ReactComponent as DocumentPreviewIcon } from './icons/document_preview.svg';
+
+const ErrorState = () => {
+  const base = `${settings.prefix}--simple-document__error-view`;
+  return (
+    <div className={base} data-testid="error_state">
+      <div className={`${base}__icon`}>
+        <DocumentPreviewIcon />
+      </div>
+      <div>
+        <h1 className={`${base}__header`}>Can’t preview document</h1>
+        <div className={`${base}__message`}>
+          Try the JSON tab for a different view of this document's data.
+        </div>
```

Comment:
The way we handle i18n messages is to allow them to be overridden by the user. They are defined as optional params, with default values.